controller/module.js

```diff
@@ -21,15 +23,22 @@ async function showModule(req, res) {
 
 async function addModule(req, res) {
   const { no, name, contents, hrsPerModule, cognitiveLevels } = req.body;
+  const isTopicValid = await isEntityIdValid(Topic, Topic);
```

Comment:
You are trying to compare `Topic` with `Topic` itself here... 
You should instead pass `contents` from `req.body`
For eg:
`isEntityValid(contents, Topic)` 
would be correct